<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ANSI-ficator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/home/ansificator10z.png</normaloff>:/home/ansificator10z.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>671</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Source Image:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="wFileIn"/>
       </item>
       <item>
        <widget class="QPushButton" name="wLoadButton">
         <property name="text">
          <string>Load!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="wBrowseButton">
         <property name="text">
          <string>Browse..</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="wSaturPlus">
         <property name="text">
          <string>Increase Contrast</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="wEffDarken">
         <property name="text">
          <string>Darken</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="wEffBrighten">
         <property name="text">
          <string>Brighten</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="wEffMono">
         <property name="text">
          <string>Monochrome</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGraphicsView" name="wGraphicsView">
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="interactive">
        <bool>false</bool>
       </property>
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string> View:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="wWindowScale">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Stretch to window size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="wANSIenable">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Display ANSI</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="wConvertButton">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Convert!</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="wStatus">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>________</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="wDumpImage">
         <property name="text">
          <string>Save Image file..</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="wDumpGFX_File">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Dump .GFX file..</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionView_GFX_file"/>
    <addaction name="separator"/>
    <addaction name="actionExitt"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="action80_x_25_16_16_color"/>
    <addaction name="action80_x_25_16_8_color"/>
   </widget>
   <widget class="QMenu" name="menuEngine">
    <property name="title">
     <string>Engine</string>
    </property>
    <addaction name="action2x2_Mosaic_comparison"/>
    <addaction name="actionPer_pixel_comparison"/>
    <addaction name="actionPer_character_comparison"/>
    <addaction name="separator"/>
    <addaction name="actionDisable_Alphanum"/>
    <addaction name="actionDisable_Sym"/>
    <addaction name="separator"/>
    <addaction name="actionAlternate_detection_palette"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionStretch_loaded_image"/>
    <addaction name="actionKeep_aspect_ratio_on_stretch"/>
    <addaction name="separator"/>
    <addaction name="actionAutomatically_Load_after_Browse"/>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuFormat"/>
   <addaction name="menuEngine"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionView_GFX_file">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>.GFX viewer..</string>
   </property>
   <property name="toolTip">
    <string>Not implemented yet.</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="action80_x_25_16_16_color">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>80 x 25 - (16 FG / 16 BG color)</string>
   </property>
  </action>
  <action name="action80_x_25_16_8_color">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>80 x 25 - (16 FG / 8 BG color)</string>
   </property>
  </action>
  <action name="actionAlternate_detection_palette">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use alternate detection palette</string>
   </property>
  </action>
  <action name="action2x2_Mosaic_comparison">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2x2 Mosaic (Per-2x2-pixel) comparison</string>
   </property>
  </action>
  <action name="actionPer_pixel_comparison">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pixel-by-pixel comparison</string>
   </property>
  </action>
  <action name="actionPer_character_comparison">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Character average comparison</string>
   </property>
  </action>
  <action name="actionDisable_Alphanum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disallow alpha-numeric characters (faster)</string>
   </property>
  </action>
  <action name="actionDisable_Sym">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Disallow symbolic characters (much faster)</string>
   </property>
  </action>
  <action name="actionDefault_settings">
   <property name="text">
    <string>Restore to default settings</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionStretch_loaded_image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Stretch image on Load</string>
   </property>
  </action>
  <action name="actionKeep_aspect_ratio_on_stretch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Keep aspect ratio on stretch</string>
   </property>
  </action>
  <action name="actionAutomatically_Load_after_Browse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Automatically Load after Browse</string>
   </property>
  </action>
  <action name="actionExitt">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
